src/emscripten-optimizer/simple_ast.h

```diff
@@ -396,19 +396,19 @@ struct Value {
         }
         os << '[';
         if (pretty) {
-          os << std::endl;
```

Comment:
For Windows, `[f]printf`, `cout/cerr` or any ostream in text/console mode will automatically expand `'\n'` to `"\r\n"`, but not in binary mode.